commands/__test__/transform.spec.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'node:path'
+import prompts from 'prompts'
+import { transform } from '../transform'
+
+describe('interative mode', () => {
+  it('Without user input', async () => {
+    prompts.inject(['magic-redirect'])
+    prompts.inject([path.resolve(process.cwd(), './transforms/__testfixtures__')])
+    const res = await transform(undefined, undefined, { dry: true, silent: true })
+
+    expect(res.ok).toBe(2)
+    expect(res.error).toBe(0)
+  }, 10000)
```

Comment:
Is this 10000 value for a test timeout? Shouldn't we mock modules we don't want to test?

Reply:
That value is to extend the time the test can run, and it was the way I found to test with the prompts module.

I don't think it's possible to mock that module yet.

Reply:
I had to extend it because the tests were always failing due to the time they took.

Reply:
I need to run this and will let you know soon. Unit tests shouldn't take that long to finish

Reply:
I have applied changes. I think this should be our direction. I hope they will work tho

Reply:
Thank you